src/cli/opts.js

```diff
@@ -62,12 +62,16 @@ export default async function parse(pkg, argv) {
   opts.include['copy-files'] = ['**/*']
   opts.exclude['copy-files'] = [...opts.include.js, ...opts.include.css, ...opts.exclude['*']]
 
+  if (process.env.NODE_ENV === 'production') {
```

Comment:
erm. what happens when `NODE_ENV=production ez-build --interactive` - shouldn't the cli switch take precedence as you are turning off interactive if opt.production on line 74.